### ISSUE TITLE
Detect magic causes sequencer adduser error

### ISSUE THREAD

**Issue by Eligarf** [Vauryx/AdvancedSpellEffects#29]
Uncaught (in promise) Error: Sequencer | Module: Advanced Spell Effects | EffectSection | _addUser - user with id "EfS5RW1o5FATIPMM" does not exist!
[Detected 2 packages: sequencer, advancedspelleffects]
    at Module.throwError (assets.forge-vtt.com/bazaar/modules/sequencer/1.2.10/scripts/module/lib/lib.js:337)
    at Proxy._throwError (assets.forge-vtt.com/bazaar/modules/sequencer/1.2.10/scripts/module/sequencer.js:175)
    at Proxy._addUser (assets.forge-vtt.com/bazaar/modules/sequencer/1.2.10/scripts/module/sections/traits/users.js:8)
    at assets.forge-vtt.com/bazaar/modules/sequencer/1.2.10/scripts/module/sections/traits/users.js:43
    at Array.forEach (<anonymous>)
    at Proxy.forUsers (assets.forge-vtt.com/bazaar/modules/sequencer/1.2.10/scripts/module/sections/traits/users.js:43)
    at Function.activateDetectMagic (assets.forge-vtt.com/bazaar/modules/advancedspelleffects/0.7.0.1/scripts/spells/detectMagic.js:151)
    at async Function.handleASE (assets.forge-vtt.com/bazaar/modules/advancedspelleffects/0.7.0.1/scripts/ASEHandler.js:41)

Running as the GM, I add the "magical, evocation, blue" setting to the tag of a token and cast the ASE version of Detect Magic on another 50 feet away. I then move the caster token to within 30' and this error pops up.

![image](https://user-images.githubusercontent.com/16523503/142441275-78bdeb77-de9f-42aa-838e-9341a21e3d0c.png)

ASE 0.7.0.1
Sequencer 1.2.10

**Comment by Vauryx** [comment]
Sorry about the late response - is this issue still happening with ASE and Sequencer updated?

**Comment by Eligarf** [comment]
Yes, I just tried it today with ASE 0.8.0.3 and Sequencer 1.3.4

**Comment by Vauryx** [comment]
I've tried a bunch of times and have not been able to recreate your error. It feels like it may be a weird user account in your world. Could you try on a fresh world with only the required modules installed and see if the issue is persisting there?

**Comment by Eligarf** [comment]
I built a new world locally and on forge, then enabled the following modules:

`Active Modules:
----------
Active-Auras v0.3.08;
Advanced Macros v1.12;
Advanced Spell Effects v0.8.0.3;
Changelogs v0.5;
Dynamic Active Effects SRD v6.0.01;
Dynamic effects using Active Effects v0.8.70;
Forien's Copy Environment v2.0.6;
JB2A - Patreon Complete Collection v0.3.2;
libWrapper v1.11.0.0;
Midi QOL v0.8.90;
Midi SRD v0.1.03;
Sequencer v1.3.5;
Simple Calendar vv1.3.75;
socketlib v1.0.9;
Tagger v1.2.1;
Tidy UI - Game Settings v0.1.28;
Times Up v0.8.26;
Warp Gate v1.10.1;
`

Both worked as expected. I then joined my game as a player, thinking that option on forge might cause the issue, but it still worked as expected. I then dropped all the modules in a different test world on forge that had exhibited the problem, it still shows the problem with this minimum set - no other players have ever joined this.  I looked at the game.users variable, it shows only the one user, but still gives this different stack on error:
`lib.js:358 Uncaught (in promise) Error: Sequencer | Module: Advanced Spell Effects | Effect | _addUser - user with id "WUaYr4RvyfC5e49f" does not exist!
[Detected 2 packages: sequencer, advancedspelleffects]
    at Module.throwError (lib.js:358)
    at Proxy._throwError (sequencer.js:175)
    at Proxy._addUser (users.js:8)
    at users.js:43
    at Array.forEach (<anonymous>)
    at Proxy.forUsers (users.js:43)
    at Function.handleConcentration (detectMagic.js:218)`

Not sure what I can even begin to do about this.
